docs/networking/vpn/connect-two-servers-using-wireguard.md

```diff
@@ -0,0 +1,96 @@
+---
+author:
+  name: Florian Brandstetter
+  email: me@florianb.space
+description: 'Connect two servers securely using WireGuard as VPN'
+keywords: 'networking,vpn, wireguard,debian,ubuntu'
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+modified: Thursday, August 25th, 2016
+modified_by:
+  name: Floria Brandstetter
+published: 'Thursday, August 25th, 2016'
+title: 'Secure Communications with OpenVPN on Ubuntu 12.04 (Precise) and Debian 7'
+external_resources:
+ - '[WireGuard Website](https://www.wireguard.io/)'
+---
+
+WireGuard is an extremely simple and fast VPN that utilizes state-of-the-art cryptography for highest security. The aim of the project is to be faster, simpler and more useful than IPSec. Currently, WireGuard is still in heavy development, altough it might already be regarded as the most secure and simplest VPN solution in the industry. WireGuard is useful for you if you want to exchange data securely between two servers.
+
+Before installing WireGuard, we assume that you have followed our [Getting Started Guide](/docs/getting-started/). If you're new to Linux server administration you may be interested in our [Introduction to Linux Concepts Guide](/docs/tools-reference/introduction-to-linux-concepts), [Beginner's Guide](/docs/beginners-guide/) and [Administration Basics Guide](/docs/using-linux/administration-basics). If you're concerned about securing on your Linode, you might be interested in our [Security Basics](/docs/security/basics) article as well.
+
+## Requirements
+This guide assumes that you're running the latest version of the Linux kernel, because WireGuard requires a kernel older than 4.1 in order to function correctly.
```

Comment:
Hello,
I am very sorry. By this, I meant the kernel should be newer than 4.1!

I'll try to fix that right now, sorry! :-(